src/Components/Reviews/FullReview.jsx

```diff
@@ -0,0 +1,35 @@
+import { useParams } from "react-router-dom";
+import { useState, useEffect } from "react";
+import axios from "axios";
+
+export default function FullReview() {
+  const [review, setReview] = useState({});
+  const { review_id } = useParams();
+
+  useEffect(() => {
+    axios
+      .get(`https://pde-games.herokuapp.com/api/reviews/${review_id}`)
+      .then((res) => {
+        setReview(res.data.review);
+      });
+  }, [review_id]);
+
+  console.log(review);
+  return (
+    <div className="full-review">
+      <h3 className="full-title">{review.title}</h3>
+      <div className="full-img-container">
+        <img
```

Comment:
You have an img inside `<div className="full-img-container">`. If the image is just going to take up the whole space you can get rid of the div completely.